libp2p/connectionMonitor/libp2pConnectionMonitorSimple.go

```diff
@@ -153,12 +206,50 @@ func (lcms *libp2pConnectionMonitorSimple) ThresholdMinConnectedPeers() int {
 	return lcms.thresholdMinConnectedPeers
 }
 
+// SetPeerDenialEvaluator sets the handler that is able to tell if a peer can connect to self or not (is or not blacklisted)
+func (lcms *libp2pConnectionMonitorSimple) SetPeerDenialEvaluator(handler p2p.PeerDenialEvaluator) error {
```

Comment:
we inject the PeerDenialEvaluator into the constructor. Why do we need this set method?
LE: I've saw this on the netMessenger:
```GO
PeerDenialEvaluator: &disabled.PeerDenialEvaluator{},
```
We should remove this from the arguments list and rely only on the set/get methods

Reply:
we have the setter as this component is created with network components among with antiflood componets which are later on used to create the peerDenialEvaluator. removed it from ctor as suggested